packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.js

```diff
@@ -60,17 +61,20 @@ const PaymentInformation = props => {
     const paymentInformation = doneEditing ? (
         <Summary onEdit={showEditModal} />
     ) : (
-        <PaymentMethods
-            onPaymentError={handlePaymentError}
-            onPaymentSuccess={handlePaymentSuccess}
-            resetShouldSubmit={resetShouldSubmit}
-            shouldSubmit={shouldSubmit}
-        />
+        <Form>
+            <PaymentMethods
+                onPaymentError={handlePaymentError}
+                onPaymentSuccess={handlePaymentSuccess}
+                resetShouldSubmit={resetShouldSubmit}
+                shouldSubmit={shouldSubmit}
+            />
+        </Form>
     );
 
-    const editModal = isEditModalActive ? (
-        <EditModal onClose={hideEditModal} />
-    ) : null;
+    const editModal =
+        doneEditing && isUpdateMode ? (
+            <EditModal onClose={hideEditModal} isOpen={isUpdateMode} />
```

Comment:
Conditionally rendering the `Dialog` component results in a loss of the smooth transition. See how the shipping information and shipping method modals smoothly open whereas the payment information modal just appears:

[![Image from Gyazo](https://i.gyazo.com/7bd027b70304ffe43ac89088c357a076.gif)](https://gyazo.com/7bd027b70304ffe43ac89088c357a076)

Reply:
I use Dialog the same with others by I don't know why the transation lost.

Reply:
> Conditionally rendering the Dialog component results in a loss of the smooth transition

:D When the condition is `true` the entire component is mounted in the "open" state, which is why it just "appears"